test/components/views/rooms/MemberTile-test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2023 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * /
+ */
+
+import React from "react";
+import { render, screen, waitFor } from "@testing-library/react";
+import { MatrixClient, RoomMember, Device } from "matrix-js-sdk/src/matrix";
+import { UserVerificationStatus, DeviceVerificationStatus } from "matrix-js-sdk/src/crypto-api";
+
+import * as TestUtils from "../../../test-utils";
+import MemberTile from "../../../../src/components/views/rooms/MemberTile";
+
+describe("MemberTile", () => {
+    let matrixClient: MatrixClient;
+    let member: RoomMember;
+
+    beforeEach(() => {
+        matrixClient = TestUtils.stubClient();
+        jest.spyOn(matrixClient, "isRoomEncrypted").mockReturnValue(true);
```

Comment:
I don't think we need to use `spyOn` here:

```suggestion
        mocked(matrixClient).isRoomEncrypted.mockReturnValue(true);
```

Likewise for the other places you've used `spyOn` below.